Dockerfile

```diff
@@ -78,14 +85,34 @@ RUN set -eux; \
 			--build="$gnuArch" \
 			--libdir="$TOMCAT_NATIVE_LIBDIR" \
 			--prefix="$CATALINA_HOME" \
+			--with-apr="$(which apr-1-config)" \
 			--with-ssl=yes; \
 		make -j "$(nproc)"; \
 		make install; \
 	); \
 	rm -rf "$nativeBuildDir"; \
 	rm bin/tomcat-native.tar.gz; \
+	\
+# sh removes env vars it doesn't support (ones with periods)
+# https://github.com/docker-library/tomcat/issues/77
 	find ./bin/ -name '*.sh' -exec sed -ri 's|^#!/bin/sh$|#!/usr/bin/env bash|' '{}' +
 
+# DEPLOY-387
+# Remove RPMs installed above (except apr, which has been linked again)
+RUN rpm -e mpfr libmpc libmnl libnfnetlink libnetfilter_conntrack iptables iproute \
+    apr-util apr-devel cpp libdb-devel m4 groff-base perl-parent perl-HTTP-Tiny perl-podlators \
+    perl-Pod-Perldoc perl-Text-ParseWords perl-Pod-Escapes perl-Encode perl-Pod-Usage perl-macros \
+    perl-libs perl-Socket perl-Time-HiRes perl-Exporter perl-constant perl-Filter perl-Carp \
+    perl-Storable perl-PathTools perl-Scalar-List-Utils perl-Time-Local perl-File-Temp perl-File-Path \
+    perl-threads-shared perl-threads perl-Pod-Simple perl-Getopt-Long perl perl-Test-Harness \
+    perl-Thread-Queue perl-Data-Dumper autoconf libkadm5 libgomp sysvinit-tools initscripts cyrus-sasl \
+    cyrus-sasl-devel openldap-devel make kernel-headers glibc-headers glibc-devel libsepol-devel \
+    pcre-devel libselinux-devel libcom_err-devel libverto-devel expat-devel keyutils-libs-devel \
+    krb5-devel zlib-devel openssl-devel apr-util-devel gcc openssl automake wget
+
+# DEPLOY-387
+RUN yum clean all
```

Comment:
We could add this to the previous RUN command to minimize layers, but it's probably fine for now.